pkg/cri/server/sandbox_run.go

```diff
@@ -163,8 +165,10 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		if err := c.setupPodNetwork(ctx, &sandbox); err != nil {
```

Comment:
FYI.. the long pole for creating the pod, by far, appears to be in cni setup:
https://github.com/containerd/go-cni/blob/main/cni.go#L142-L155
on my computer a default network setup appears to be taking 1.5 seconds

DEBU[2021-11-01T09:31:42.207910789-05:00] begin cni setup podsandboxid=9b068ec35dfaffe5e92287db5d24731a30569d13ab88addfeb07361956d407e4
DEBU[2021-11-01T09:31:43.684393004-05:00] cni result: {...} podsandboxid=9b068ec35dfaffe5e92287db5d24731a30569d13ab88addfeb07361956d407e4